### PR TITLE
Added document parties in POST booking

### PR DESCRIPTION
**Pull request by douradofrazer**
Resolves:
https://dcsa.atlassian.net/browse/DDT-543

### CODE REVIEW

File: src/main/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTO.java [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
   private PartyTO party;
 
   @NotNull(message = "PartyFunction is required.")
   private PartyFunction partyFunction;
 
   private List<String> displayedAddress;
 
-  @NotNull(message = "PartyContactDetails is required.")
-  @NotEmpty(message = "PartyContactDetails is required.")
-  private List<PartyContactDetailsTO> partyContactDetails;
```
Comment (pimg) [inline src/main/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTO.java:25]: why is this removed? The OAS still has PartyContactDetails

File: src/test/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTOTest.java [hunk 2]
```diff
@@ -19,20 +20,22 @@
 class DocumentPartyTOTest {
```
Comment (pimg) [inline src/test/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTOTest.java:20]: missing test case for displayed address larger then 250 characters

**Review by pimg — CHANGES_REQUESTED** [review]
made two comments

File: src/main/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTO.java (hunk 1 above)
Comment (douradofrazer) [reply src/main/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTO.java:25]: It's moved to PartyTO as it was more relevant to a party object as discussed with Henrik.

File: src/test/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTOTest.java (hunk 2 above)
Comment (douradofrazer) [reply src/test/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTOTest.java:20]: It's already there, line 55
`
  @Test
  @DisplayName(
      "DocumentPartyTO should throw error if displayedAddress length exceeds max size of 250.")
`

**Review by pimg — APPROVED** [review]

**Review by pimg — APPROVED** [review]
